redwood-snapshot-testing/build.gradle

```diff
@@ -0,0 +1,28 @@
+import static app.cash.redwood.buildsupport.TargetGroup.CommonWithAndroid
+
+apply plugin: 'org.jetbrains.kotlin.plugin.compose'
+
+redwoodBuild {
+  targets(CommonWithAndroid)
+}
+
+kotlin {
+  sourceSets {
+    commonMain {
+      dependencies {
+        api libs.jetbrains.compose.foundation
+        api libs.kotlin.test
+        implementation libs.assertk
+      }
+    }
+    androidMain {
+      dependencies {
+        api libs.paparazzi
```

Comment:
Not the Paparazzi plugin, 'cause this is a multiplatform library and it isn’t expecting that